app/controllers/resources_controller.rb

```diff
@@ -1,14 +1,22 @@
 class ResourcesController < ApplicationController
   def index
     category_id = params[:category_id]
+    latitude = params[:latitude]
+    longitude = params[:longitude]
+
     result = if category_id
                resources.joins(:categories).where('categories.id' => category_id)
              end
 
     if result
+      if latitude and longitude
```

Comment:
`&&` is usually the intended operator here. http://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby